charts/growthbook/templates/secrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: growthbook-secrets
+type: Opaque
+data:
+  # generate 32 chars long random string, base64 encode it and then double-quote the result string.
+  jwt-secret: {{ randAlphaNum 32 | b64enc | quote }}
+  encryption-key: {{ randAlphaNum 32 | b64enc | quote }}
```

Comment:
Hmm I wouldn't do that... they will be re-generated on every upgrade (I think?)!

Reply:
True, but I'm not sure about the behaviour of the appplication... If it starts encrypting stuff with one key, and then on next upgrade tries to decrypt it with a different key, it probably won't work...

I know some charts have figured out a way to autogenerate a new value (if none is provided), but _only on the first install_. I don't fully understand how it works, but it proves it's doable... though I'm not sure it's worth the effort. See [this](https://github.com/bitnami/charts/blob/master/bitnami/postgresql/templates/secrets.yaml#L17)